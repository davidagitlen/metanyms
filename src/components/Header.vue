<template>
  <header id='header'>
    <h1>{Metanyms}</h1>
    <p>Your One-Stop Synonym Shop!</p>
  </header>
</template>

<style scoped>
  h1 {
    font-size: 4.5em;
  }
  p {
    font-size: 2em;
  }
  #header {
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    align-items: center;
    height: 20vh;
    color: #3030FF;
    background-color: #3DF4DF;
  }
</style>