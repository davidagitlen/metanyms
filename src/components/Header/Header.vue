<template>
  <header id='header'>
    <h1>{Metanyms}</h1>
    <p>Your One-Stop Synonym Shop!</p>
  </header>
</template>

<style scoped>
  h1 {
    font-size: 4.5em;
  }
  p {
    font-size: 1.5em;
    margin: 7.5px 0px 0px 100px;
  }
  #header {
    align-items: center;
    height: 16vh;
    min-height: 125px;
    padding: 20px 0px 0px 30px;
    color: #3030FF;
    background-color: #00E8CC;
  }
</style>