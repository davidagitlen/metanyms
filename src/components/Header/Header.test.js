import { mount } from '@vue/test-utils';
import Header from './Header.vue';

describe('Header', () => {

  test('renders correctly', () => {
    const wrapper = mount(Header);
    expect(wrapper.element).toMatchSnapshot();
  })

})