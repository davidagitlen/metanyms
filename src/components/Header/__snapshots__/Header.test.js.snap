// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  id="header"
>
  <h1>
    {Metanyms}
  </h1>
   
  <p>
    Your One-Stop Synonym Shop!
  </p>
</header>
`;
